index.html

```diff
@@ -2073,6 +2073,42 @@ <h3>Transformations</h3>
         </p>
       </section>
 
+      <section>
+        <h3>Data Opacity</h3>
+
+        <p>
+The inspectability of application data has effects on system efficiency and
+developer productivity. When cryptographically protected application data, such
+as base-encoded binary data, is not easily processed by application subsystems,
+such as databases, it increases the effort of working with with the
+cryptographically protected information. For example, a cryptographically
+protected payload that can be natively stored and indexed by a database will
+result in a simpler system. Similarly, a cryptographically protected payload
```

Comment:
Not talking about the "unseen" complexities here by only mentioning a "simpler system" may not address people's concerns -- we'll see. Some of the "unseen" benefits include not losing integrity proofs when data is transformed:

* allowing application developers to avoid creating complex systems that duplicate data (having one copy that is data integrity protected and another that is essentially not... but is presumed to have been checked and is what the application actually runs on)
* allowing application developers to avoid each creating their own solutions (which may differ and therefore generate more work in aggregate) to solve the impedance mismatch of using the data vs. maintaining one or more data integrity proofs (with multiple proofs perhaps implying multiple nested copies of the data).

Reply:
It may also be worth mentioning that there are encrypted-at-rest (EAR) DBMS out there (Virtuoso is one), such that the DBMS natively stores and indexes the data, but it's not extractable from that storage without the relevant key(s).  This can result in a deceptively simple system, i.e., a system which appears simple to the casual observer, but is significantly more complex below the surface.

Reply:
@dlongley wrote:
> Not talking about the "unseen" complexities here by only mentioning a "simpler system" may not address people's concerns

Added language in 60a61cb2124da4822537cc7f74cb9c664159ca14 to address your concern.

@TallTed wrote:
> It may also be worth mentioning that there are encrypted-at-rest (EAR) DBMS out there

Added language in 007125d7d9e4ba2dc983c3b2c1fa1481d01851dd to address your concern.